packages/circuits/test/circuits/Play.test.circom

```diff
@@ -2,4 +2,4 @@ pragma circom 2.0.0;
 
 include "../../src/proofs/Play.circom";
 
-component main {public [handRoot, newHandRoot, playedCardLeaf]} = Play(2);
\ No newline at end of file
+component main {public [handRoot, newHandRoot, saltHash, publicRandom, lastIndex, playedCardLeaf]} = Play(2);
```

Comment:
To make this private, we just need to check that `unpackedHand[lastIndex] != 0 && (unpackedHand[lastIndex+1] = 0 || lastIndex == 63))`. Annoying because of needing to iterate the whole array, so probably not worth it.